TwitterSentiment/pom.xml

```diff
@@ -47,8 +47,9 @@
 
   <repositories>
     <repository>
-      <id>apache.snapshots</id>
-      <url>https://repository.apache.org/content/repositories/snapshots</url>
+      <id>oss.sonatype</id>
+      <name>OSS Sonatype Public</name>
+      <url>http://oss.sonatype.org/content/groups/public/</url>
```

Comment:
Both Ajai and I were getting problems building some apps in cdap-apps (finding the cdap 2.8 snapshot dependencies), until we put this in.